test-harness/helpers.ts

```diff
@@ -44,15 +44,21 @@ export function parseScenarioFile(data: string): IScenarioFile {
   const env = getItem(split, 'env');
 
   if (command === void 0) {
-    if (commandWindows !== void 0 && commandUnix !== void 0) {
-      throw new Error('No ====command==== provided');
+    if (commandWindows === void 0 && commandUnix === void 0) {
+      throw new Error(
+        'When no ====command==== is provided, both ====command-nix==== and ====command-win==== should be defined',
```

Comment:
@P0lip Updated with a different approach

Reply:
Well, I actually like the change, since it might be easy to forget to specify a Windows/Unix counterpart.
To reduce the severity of that decision, we could potentially issue a warning instead of error out the whole suite.